File: KUPFAPI/Models/FormTitleDt.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

#nullable disable

namespace API.Models
{
    public partial class FormTitleDt
    { 
        [Column("TenentId")]
        public int TenentId { get; set; }
        [Column("FormId")]
        public string FormId { get; set; }
        [Column("Language")]
        public int Language { get; set; }
        [Key]
        public string LabelId { get; set; }
        public string Title { get; set; }
        public string ArabicTitle { get; set; }
        public string Rl { get; set; }
        public string Attiribute { get; set; }
        public string Remarks { get; set; }
        public string Status { get; set; }
        
        
        
    }
}

File: AngularMain/src/app/modules/models/formTitleDt.ts
import { FormTitleHd } from "./formTitleHd";

export interface FormTitleDt {
   
    id:number;   
    tenentId: number;
    formTitleHDLanguageId:string;
    formID:string;
    language: number;
    labelId: string;
    title: string;
    arabicTitle: string;
    rl: string;
    attiribute: string;
    remarks: string;
    status: string;
    orderBy:number;
    // For Inline Edit
    isBodyEdit:boolean;
}